src/handbook/peopleops/expenses.md

```diff
@@ -78,8 +78,7 @@ or CEO, who will provision it for you.
 
 ### Workspace & Location
 
-As an all remote company we don't have offices. It is envisaged that most people will work from their home the majority of the time. Equipment & Furniture required for a home workspace should be discussed with your manager as part of the hiring process.
-The company covering certain costs, such as broadband can have tax implications depending on your local legislation. You should consult an independent advisor in your location for your particular situation.
+As a company that operates entirely remotely, we do not maintain physical office spaces. Our expectation is that the majority of employees will work from their homes. During the hiring process, it is recommended that individuals discuss the necessary equipment and furniture for their home workspace with their manager. It is ideal for remote workers to have necessary equipment, but if needed, FlowForge will cover the cost of an office chair and desk. However, please be aware that certain expenses, such as broadband costs, covered by FlowForge, might have tax implications based on local legislation. To understand the specific implications for your situation, it's advisable to consult an independent advisor familiar with your local tax laws.
```

Comment:
We should be more clear here what we mean.

Reply:
I've just made these changes on #879